src/service/vaults.ts

```diff
@@ -201,14 +204,21 @@ export const watchVaultFactory = (netconfigUrl: string) => {
       const interestRate = current.InterestRate.value;
       const liquidationPenalty = current.LiquidationPenalty.value;
       const liquidationMargin = current.LiquidationMargin.value;
+      const liquidationPadding = current.LiquidationPadding.value;
       const loanFee = current.LoanFee.value;
 
+      const minCollateralizationRatio = calculateMinimumCollateralization(
+        liquidationMargin,
+        liquidationPadding,
+      );
+
       useVaultStore.getState().setVaultGovernedParams(id, {
         debtLimit,
         interestRate,
         liquidationMargin,
         liquidationPenalty,
         loanFee,
+        minCollateralizationRatio,
```

Comment:
it's not really a governed param. `liquidationPadding` is. I guess that it's fully implied by the governed params so it makes sense to keep it here but consider giving it a distinguishing name like `inferredMinimumCollateralization`. (adding "Ratio" is Hungarian notation)

Reply:
Yea it makes more sense to calculate this upfront but I can see how it could be confusing as it's not an actual param. Renamed as suggested.